pkg/extensions/minimal.go

```diff
@@ -0,0 +1,23 @@
+// +build minimal
+
+package extensions
+
+import (
+	"time"
+
+	"github.com/anuvu/zot/pkg/log"
+	"github.com/anuvu/zot/pkg/storage"
+	"github.com/gorilla/mux"
+)
+
+// DownloadTrivyDB ...
+func DownloadTrivyDB(dbDir string, log log.Logger, updateInterval time.Duration) error {
+	return nil
+}
+
+func EnableExtension(extension *ExtensionConfig, log log.Logger, rootDir string) {
+	log.Info().Msg("given zot binary doesn't support any extensions, please build zot full binary for this feature")
+}
+
+func SetupRoutes(router *mux.Router, rootDir string, imgStore *storage.ImageStore, log log.Logger) {
+}
```

Comment:
Add a similar log saying routes were not setup because extensions are not enabled.